projects/github-actions/repo-gardening/README.md

```diff
@@ -1,17 +1,93 @@
-- WordPress.com Commit Reminder: Posts a comment on merged PRs to remind Automatticians to commit the matching WordPress.com change.
+- Assign Issues (`assignIssues`): Adds assignee for issues which are being worked on, and adds the "In Progress" label.
+- Add Milestone (`addMilestone`): Adds a valid milestone to all PRs that get merged and don't already include a milestone.
+- Check Description (`checkDescription`): Checks the contents of a PR description, and ensure it matches our recommendations.
+- Add Labels (`addLabels`): Adds labels to PRs that touch specific features.
+- Clean Labels (`cleanLabels`): Removes Status labels once a PR has been merged.
+- WordPress.com Commit Reminder (`wpcomCommitReminder`): Posts a comment on merged PRs to remind Automatticians to commit the matching WordPress.com change.
+- Notify Design (`notifyDesign`): Sends a Slack Notification to the Design team to request feedback, based on labels applied to a PR.
+
+Some of the tasks are may not satisfy your needs. If that's the case, you can use the `tasks` option to limit the action to the list of tasks you need in your repo. See the example below to find out more.
+
+## Usage
+
+### Example
+
+In the example below, we'll set up the action to run 2 tasks (`cleanLabels` and `notifyDesign`):
+
+```yml
+name: Clean Labels and notify design
+
+on:
+  # We need to listen to all these events to catch all scenarios
+  # where notifying the Design team or cleaning labels may be necessary.
+  # Refer to src/index.js to see a list of all events each task needs to be listen to.
+  pull_request:
+    types: ['closed', 'labeled']
+
+jobs:
+  repo-gardening:
+    name: 'Clean up labels, and notify Design when necessary'
+    runs-on: ubuntu-latest
+    timeout-minutes: 10  # 2021-03-12: Successful runs seem to take a few seconds, but can sometimes take a lot longer since we wait for previous runs to complete.
+
+    steps:
+     - name: Checkout
+       uses: actions/checkout@v2
+
+     - name: Setup Node
+       uses: actions/setup-node@v2
+        with:
+          node-version: 12
+
+     - name: Wait for prior instances of the workflow to finish
+       uses: softprops/turnstyle@v1
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+     - name: 'Run gardening action'
+       uses: automattic/action-repo-gardening@v1
+       with:
+          github_token: ${{ secrets.GITHUB_TOKEN }}
+          slack_token: ${{ secrets.SLACK_TOKEN }}
+          slack_design_channel: ${{ secrets.SLACK_DESIGN_CHANNEL }}
+          tasks: 'cleanLabels,notifyDesign'
+```
+
+### Inputs
+
+The action relies on 4 parameters. 
+
+- `GITHUB_TOKEN` is automatically created by GitHub to be used by the action. You do not have to deal with it.
+- `tasks` is an optional parameter you can specify when setting up the action. It allows you to run selected tasks instead of the full suite. You can find the list of the different tasks (and what event it's attached to) in `src/index.js`.
```

Comment:
```suggestion
- `tasks` allows for running selected tasks instead of the full suite. The value is a comma-separated list of task identifiers. You can find the list of the different tasks (and what event it's attached to) in `src/index.js`.
```